frontend/src/shared/components/tasque-input/input.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="input-container">
+  <label>{{inputLabel}}</label>
+  <input
+  [class]="inputClass"
+  [ngClass]="{
+    'error': inputValueisError, 
+    'search-border': inputType === 'search'}"
+    [type]="inputType" 
+    [placeholder]="inputPlaceholder"
+    [value]="inputText"
+    [style.width.px]="inputWidth"
+    [style.border-radius.px]="inputBorderRadius"
+  />
+  <svg *ngIf="inputType === 'search'" class="search-icon" width="15" height="16" viewBox="0 0 15 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <g clip-path="url(#clip0_1106_2569)">
+      <path d="M14.7187 13.5677L10.7812 9.60545C10.5937 9.41677 10.3125 9.32243 10.0312 9.32243C10.7812 8.37904 11.25 7.05828 11.25 5.73753C11.25 2.62432 8.71875 0.0771484 5.625 0.0771484C2.53125 0.0771484 0 2.62432 0 5.73753C0 8.85073 2.53125 11.3979 5.625 11.3979C6.9375 11.3979 8.25 10.9262 9.1875 10.0771C9.1875 10.3602 9.1875 10.6432 9.46875 10.8319L13.4063 14.7941C13.5938 14.9828 13.875 15.0771 14.0625 15.0771C14.25 15.0771 14.5312 14.9828 14.7187 14.7941C15.0937 14.5111 15.0937 13.9451 14.7187 13.5677ZM5.625 9.98281C3.28125 9.98281 1.40625 8.09602 1.40625 5.73753C1.40625 3.37904 3.28125 1.49224 5.625 1.49224C7.96875 1.49224 9.84375 3.37904 9.84375 5.73753C9.84375 8.09602 7.96875 9.98281 5.625 9.98281Z" fill="#CBCDD3"/>
+    </g>
+    <defs>
+      <clipPath id="clip0_1106_2569">
```

Comment:
Is there a way to use more meaningful name?

Reply:
Deleted this part